.github/workflows/check-issue-status.yml

```
@@ -0,0 +1,87 @@
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  verify-issue-status:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # hoặc dùng secrets.PROJECT_PAT nếu cần scope rộng hơn
+    steps:
+      - name: 🔍 Extract Issue Numbers from Description
+        id: extract
+        run: |
```

actionlint (1.7.7)

"github.event.pull_request.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)